src/BenchmarkDotNet.Core/Attributes/Jobs/UapJobAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+
+namespace BenchmarkDotNet.Attributes.Jobs
+{
+    public class UapJobAttribute : JobConfigBaseAttribute
+    {
+        public UapJobAttribute(string devicePortalUri, string csfrCookie, string wmidCookie, string uapBinariesPath)
```

Comment:
I plan to write separate documentation about how this supposed to work. Process of bringing it all ready not so straightforward that can be described in several lines of summary.